l10n_it_edi_related_document_type/models/account_move_line.py

```diff
@@ -0,0 +1,13 @@
+from odoo import fields, models
+
+
+class AccountInvoiceLine(models.Model):
```

Comment:
```suggestion
class AccountMoveLine(models.Model):
```
Solo per coerenza con il model, ma se vuoi lascia pure com'è

Reply:
Questo codice non esiste più quindi 👍